include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -180,6 +183,9 @@ ORT_API_STATUS(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_function
                _In_ const char* logid,
                _Outptr_ OrtEnv** out);
 
+//Don't call OrtReleaseEnv from Dllmain
```

Comment:
Put this comment next to "ORT_RUNTIME_CLASS(Env);"?

I'm thinking of avoiding using the macro here to declare all the "OrtRelease***" APIs, which is not that straightforward and clear to users (given we don't have full documentation right now).